tests/settings/modify_setting_manage_dhcp_test.py

```diff
@@ -0,0 +1,72 @@
+import pytest
```

Comment:
Please move this file into the XML-RPC test directory. It doesn't validate the settings directly but only the XML-RPC interface that we use.